pkg/apis/system/v1beta1/types.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const (
+	BundleNone       = "none"
+	BundleCollecting = "collecting"
+	BundleCollected  = "collected"
```

Comment:
So they are enum of Bundle.Status?
By convention, it should be declared like this:
```
type BundleStatus string
const (
	BundleStatusNone       = "None"
	BundleStatusCollecting = "Collecting"
	BundleStatusCollected  = "Collected"
)
```
See https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L2268 for more examples

Reply:
Make sense, updated.